templates/.rubocop.yml

```diff
@@ -29,9 +29,7 @@ Metrics/BlockLength:
   - 'spec/**/*.rb'
 
 RSpec/MultipleExpectations:
-  Exclude:
-    - 'spec/system/**/*'
-    - 'spec/requests/**/*'
+  Enabled: false
```

Comment:
can you give me some background? i like the fact, that we try to follow that principle.

Reply:
it goes together with the aggregate_failures. This cop checks that you have only one `expect` or use `aggregate_failures` when having multiple expectations. Since we enabled aggregate_failures globally, this check is not needed anymore.